### ISSUE TITLE
what is my_tc ?

### ISSUE THREAD

**Issue by ucas010** [pytorch/serve#2057]
### 🐛 Describe the bug

torchserve --start --model-store model_store --models my_tc=BERTTokenClassification.mar --ncs
curl -X POST http://127.0.0.1:8080/predictions/my_tc -T Token_classification_artifacts/sample_text_captum_input.txt

### Error logs

2023-01-05T15:51:41,260 [INFO ] W-9001-my_tc_1.0-stdout MODEL_LOG - model_name: my_tc, batchSize: 1
2023-01-05T15:51:41,628 [INFO ] W-9003-my_tc_1.0-stdout MODEL_LOG - Listening on port: /tmp/.ts.sock.9003
2023-01-05T15:51:41,633 [INFO ] W-9003-my_tc_1.0-stdout MODEL_LOG - Successfully loaded /data//python3.9/site-packages/ts/configs/metrics.yaml.

So **model_name** is my_tc ? not BERTTokenClassification

### Installation instructions

conda install -c pytorch torchserve torch-model-archiver torch-workflow-archiver

### Model Packaing

torch-model-archiver --model-name BERTTokenClassification --version 1.0 --serialized-file Transformer_model/pytorch_model.bin --handler ./Transformer_handler_generalized.py --extra-files "Transformer_model/config.json,./setup_config.json,./Token_classification_artifacts/index_to_name.json"

model_name is what ?

### Versions

------------------------------------------------------------------------------------------
Environment headers
------------------------------------------------------------------------------------------
Torchserve branch: 

torchserve==0.7.0b20221212
torch-model-archiver==0.7.0b20221212

Python version: 3.9 (64-bit runtime)
Python executable: /data/python

Versions of relevant python libraries:
captum==0.6.0
future==0.18.2
numpy==1.24.1
nvgpu==0.9.0
psutil==5.9.4
requests==2.28.1
sentence-transformers==2.2.2
sentencepiece==0.1.97
torch==1.13.1+cu116
torch-model-archiver==0.7.0b20221212
torch-workflow-archiver==0.2.6b20221212
torchaudio==0.13.1+cu116
torchserve==0.7.0b20221212
torchvision==0.14.1+cu116
transformers==4.26.0.dev0
wheel==0.37.1
torch==1.13.1+cu116
**Warning: torchtext not present ..
torchvision==0.14.1+cu116
torchaudio==0.13.1+cu116

Java Version:

OS: CentOS Linux release 7.9.2009 (Core)
GCC version: (GCC) 4.8.5 20150623 (Red Hat 4.8.5-44)
Clang version: N/A
CMake version: version 2.8.12.2

Is CUDA available: Yes
CUDA runtime version: 11.6.124
GPU models and configuration: 
GPU 0: Tesla T4
GPU 1: Tesla T4
GPU 2: Tesla T4
Nvidia driver version: 510.108.03
cuDNN version: Probably one of the following:
/usr/local/cuda-11.6/targets/x86_64-linux/lib/libcudnn.so.8
/usr/local/cuda-11.6/targets/x86_64-linux/lib/libcudnn_adv_infer.so.8
/usr/local/cuda-11.6/targets/x86_64-linux/lib/libcudnn_adv_train.so.8
/usr/local/cuda-11.6/targets/x86_64-linux/lib/libcudnn_cnn_infer.so.8
/usr/local/cuda-11.6/targets/x86_64-linux/lib/libcudnn_cnn_train.so.8

### Repro instructions

just want to know what is my_tc

**Comment by msaroufim** [comment]
Yeah the format is model_name=path_to_mar_file https://github.com/pytorch/serve/blob/master/ts/arg_parser.py#L53

Would you suggest we change this behavior? I've heard this ask before here https://github.com/pytorch/serve/issues/2037
